aidial_adapter_bedrock/llm/model/claude/v3/tools.py

```diff
@@ -36,12 +44,46 @@ class ToolsMode(Enum):
 
 
 def process_tools_block(
-    consumer: Consumer, block: ToolUseBlock, tools_mode: ToolsMode
+    consumer: Consumer, block: ToolUseBlock, tools_mode: ToolsMode | None
 ):
     match tools_mode:
         case ToolsMode.NATIVE_TOOLS:
             consumer.create_function_tool_call(to_dial_tool_call(block))
         case ToolsMode.FUNCTION_EMULATION:
             consumer.create_function_call(to_dial_function_call(block))
         case _:
-            assert_never(tools_mode)
+            raise Exception(f"Invalid tools mode {tools_mode} during tool use!")
```

Comment:
Same comment as for `to_dial_finish_reason` method

Reply:
fixed